packages/records/lib/models/records.ts

```diff
@@ -263,7 +263,7 @@ export async function upsert({
                             connection_id: connectionId,
                             model,
                             environment_id: environmentId,
-                            count: Math.max(0, delta)
```

Comment:
unrelated but max was only useful when the counts were not populated yet for historic data. removing it since it doesn't bring anything